charts/rstudio/templates/ingress.yml

```diff
@@ -2,11 +2,14 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: r-studio
+  labels:
+    app: r-studio
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
   annotations:
     kubernetes.io/ingress.class: "nginx"
 spec:
   rules:
-  - host: {{.Username}}-rstudio.{{.ToolsDomain}}
+  - host: {{ .Values.Username }}-rstudio.{{ .Values.ToolsDomain }}
```

Comment:
Also, I think this is actually `$USER.rstudio.$TOOLS_DOMAIN` (dot instead of dash).

But how did this work before?